pkg/manager/member/tidbcluster_status_manager.go

```diff
@@ -135,6 +135,7 @@ func (m *TidbClusterStatusManager) syncDashboardMetricStorage(tc *v1alpha1.TidbC
 }
 
 // ref https://github.com/pingcap/tidb/blob/36b04d1aa01db722b3f07af759168c6b8da33801/domain/infosync/info.go#L72
+// search `TopologyInformationPath` about how theey with 'ttl' and 'info' key is write in that file.
```

Comment:
typo for `theey`?

Reply:
fix in https://github.com/pingcap/tidb-operator/pull/3929/commits/08a6ee483dade1d8947c187fa8f9ec1db921a17b